### ISSUE TITLE
Undocumented what shortcuts are affected by Paredit Hijack setting

### ISSUE THREAD

**Issue by skylize** [BetterThanTomorrow/calva#2136]
The setting `Calva > Paredit: Hijack VSCode Defaults` overrides "more" built-in shortcuts. What is "more"? How many settings? Does this have a severe impact on usability or just enable a few convenience features? 

No way to judge the value of enabling/disabling this setting because neither the description nor documentation provides the slightest insight into which shortcuts are affected.

---

I also wonder whether this setting should even exist. Perhaps one setting or the other should just be chosen for everyone, with any deviations being moved to the docs as suggestions for customization.

I can easily imagine possible usefulness depending on what shortcuts are affected. However, considering how long the setting has existed without the documentation needed to use it, and the lack of users begging for the type of behavior it already potentially provides for, it seems likely this only adds unnecessary maintenance complexity and noise in the settings menu.

**Comment by PEZ** [comment]
Yes, we should document this. Though, the original intention with the setting was to provide for a way to test if some keybinding can be hijacked without causing too much disruption for people. But instead we have gotten stuck in a ”contract” about the defaults, where someone might have opted out. Even so, I think that visiting the original intention and move some of the currently ”hijacked” shortcuts to the default configured group makes sense. People can always rebind.

> the lack of users begging for the type of behavior it already potentially provides for /.../ it seems likely this only adds unnecessary maintenance complexity

Not sure I follow here. It is on by default, so that probably explains why people do not ask for the behaviour so often.

**Comment by skylize** [comment]
> Not sure I follow here. It is on by default, so that probably explains why people do not ask for the behaviour so often.

What I mean is:

(I could certainly have missed it, but) to my knowledge nobody has been making any requests to toggle off some large subset of Calva shortcuts through a single setting. That is the use case where a setting like this would provide user-facing value. So for the user case, it seems like extra trouble to keep a setting like this in existence at all.

Regardless, if the setting provides Calva-
Dev-facing value, my comment about maintenance cost is meaningless.

**Comment by PEZ** [comment]
Calva already has a toggle for switching off all Paredit shortcuts. On request.
